features/step_definitions/metadata.js

```diff
@@ -33,31 +33,24 @@ module.exports = function server() {
             throw new Error('jobName is neither BAR or BAZ');
         }
 
-        request({
+        return request({
             uri: `${this.instance}/${this.namespace}/jobs/${jobId}/builds`,
             method: 'GET',
             json: true
         }).then((response) => {
             this.buildId = response.body[0].id;
-            this.meta = response.body.meta;
 
             Assert.equal(response.statusCode, 200);
         });
     });
 
-    this.Then(/^add the { "foo": (.*) } to metadata in the "main" build container$/, () => null);
+    this.Then(/^add the { "foo": (?:.*) } to metadata in the "main" build container$/, () => null);
```

Comment:
If `?:` is not used, we have to make coding like this. It is mandatory to give an argument `value`.
```
this.Then(/^add the { "foo": (.*) } to metadata in the "main" build container$/, (value) => {
  // Must do something with ${value}
});
```
There is no way to do a good test using "value", so I think that "value" can be discarded.

--
Or simply, such coding will work. Is this better?
```
this.Then(/^add the { "foo": "foobar" } to metadata in the "main" build container$/, () => null);
 ```